plugin/src/test/sc/plugin2021/BoardTest.kt

```diff
@@ -1,68 +1,86 @@
 package sc.plugin2021
 
 import io.kotest.matchers.shouldBe
-import io.kotest.core.spec.style.StringSpec
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldNotBe
 import io.kotest.matchers.types.shouldNotBeSameInstanceAs
-import sc.plugin2021.util.Constants
 
-class BoardTest : StringSpec({
-    "Test correct board initialisation" {
-        val board = Board()
-        
-        board shouldNotBeSameInstanceAs Board()
-        board shouldBe Board()
-    
-        for (x in 0 until Constants.BOARD_SIZE) {
-            for (y in 0 until Constants.BOARD_SIZE) {
-                board[x, y] shouldNotBeSameInstanceAs Field(Coordinates(x, y), FieldContent.EMPTY)
-                board[x, y] shouldBe Field(Coordinates(x, y), FieldContent.EMPTY)
+class BoardTest : WordSpec({
+    val changingFields = setOf(
+            Field(Coordinates(2, 3), FieldContent.YELLOW),
+            Field(Coordinates(19, 2), FieldContent.YELLOW),
+            Field(Coordinates(8, 3), FieldContent.YELLOW),
+            Field(Coordinates(4, 9), FieldContent.YELLOW)
+    )
+    "Boards " When {
+        var one: Board
+        var two: Board
+        "constructed" Should {
+            one = Board()
+            two = Board()
```

Comment:
when you use a proper `isolationMode` like I did in GameTest you can move that into common code

Reply:
I'm aware, this was more to get into the testing style. The whole "When ... constructed" is superfluous